libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -314,7 +311,32 @@ string ABIFunctions::conversionFunction(Type const& _from, Type const& _to)
 			}
 			else if (toCategory == Type::Category::FixedPoint)
 			{
-				solUnimplemented("Not yet implemented - FixedPointType.");
+				FixedPointType const& toFixedPointType = dynamic_cast<FixedPointType const&>(_to);
+				if (fromCategory == Type::Category::Integer)
+				{
+					body =
```

Comment:
I wonder if we should add an assertion here for the type width being appropriate?

Reply:
Explicit conversions don't seem to check whether the target is of equal or exceeding width.

Github is sabotaging my review as it is constantly timing out :(